task/orka-teardown/0.2/tests/resources.yaml

```
@@ -0,0 +1,49 @@
+# Secrets set and used for local test instance.
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: orka-tekton-config
+data:
+  ORKA_API: http://localhost:8080
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: orka-creds
+type: Opaque
+stringData:
+  email: tekton-svc@macstadium.com
+  password: p@ssw0rd
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: orka-svc
+  namespace: orka-teardown-0-2
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: orka-runner-teardown
```

this ClusterRole can probably clash with version 0.1's ClusterRole, hence we'll have to update the name here 😅 
```suggestion
  name: orka-runner-teardown-0-2
```